core/storage/topic/gae_models_test.py

```diff
@@ -0,0 +1,147 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test for Topic model."""
+
+from constants import constants
+from core.domain import subtopic_page_domain
+from core.domain import subtopic_page_services
+from core.domain import topic_domain
+from core.domain import topic_services
+from core.platform import models
+from core.tests import test_utils
+import feconf
+
+(topic_models,) = models.Registry.import_models([models.NAMES.topic])
+
+
+class TopicModelUnitTest(test_utils.GenericTestBase):
```

Comment:
Test --> Tests

Reply:
What should I change class name or docstring typo?

Reply:
In case that you mean about the 'Test for model class' typo. It's done (https://github.com/oppia/oppia/pull/5689/commits/eec19ef8b88a7f5eae3c294f05dbe9258baf8006)

Reply:
Usually comments refer to the line that they are made underneath. In this case, the comment is referring to the class name in line 31.

Reply:
Ok, so it's done (commit: https://github.com/oppia/oppia/pull/5689/commits/225931455c9ce88b9f599c5e43ee5cdba44e2954)